Assets.md

```diff
@@ -4,6 +4,9 @@ We use asset catalogs to organize our application’s assets.
 
 ### Key Considerations
 * Organize assets using folders inside asset catalogs.
-* Whenever possible, use images and colors from asset catalogs using Interface Builder.
-* When assets cannot be used from Interface Builder (i.e. the assets to use from the catalog are not known at compile time), make use of code generation solutions such as [SwiftGen](https://github.com/SwiftGen/SwiftGen#asset-catalog) to provide non-optional constants for assets.
+* All colors and images that are known at compile time should exist in asset catalogs. 
+* Asset catalog colors should come from a project’s design system for project-wide consistency, ease of small tweaks, and to assist in supporting color-related features like dark mode. Avoid adding colors for specific use cases without verifying it with the project’s designers and updating the design system.
+* Whenever possible, set images from asset catalogs in Interface Builder. 
+* Always set colors from asset catalogs in code, never in Interface Builder.
+* When assets cannot be used from Interface Builder (i.e. images not available at compile time or colors), make use of code generation solutions such as [SwiftGen](https://github.com/SwiftGen/SwiftGen#asset-catalog) to provide non-optional constants for assets.
```

Comment:
The parenthetical here confused me. Maybe reword?

Reply:
Simplified